src/root/static/js/common.js

```diff
@@ -50,25 +50,13 @@ $(document).ready(function() {
 
     $(".hydra-popover").popover({});
 
-    $(function() {
-        if (window.location.hash) {
-            $(".nav-tabs a[href='" + window.location.hash + "']").tab('show');
-        }
-
-        /* If no tab is active, show the first one. */
```

Comment:
Can't see what this was needed for, so just removed it.

Reply:
This was used to open the first available tab in the `eval` view

 - https://hydra.sharcnet.ca/eval/575
 - https://hydra.nixos.org/eval/1656754

In that first link, the *Still succeeding jobs* tab should get activated.